gapis/trace/tracer/tracer.go

```diff
@@ -84,9 +84,15 @@ func LayersFromOptions(ctx context.Context, o *service.TraceOptions) []string {
 	if o.PerfettoConfig == nil {
 		return ret
 	}
+Outer:
 	for _, x := range o.PerfettoConfig.GetDataSources() {
 		if layer, err := layout.LayerFromDataSource(x.GetConfig().GetName()); err == nil {
 			if _, err := layout.LibraryFromLayerName(layer); err == nil {
+				for _, l := range ret {
+					if l == layer {
+						continue Outer
+					}
+				}
```

Comment:
Maube put them in a map and then return the map's keys? I guess N^2 here is ok, but...

Reply:
I figure with the typical number of layers we have, the map might make it worse.  But I don't mind changing to map either.

Reply:
Done